lib/dao.js

```diff
@@ -1659,7 +1669,7 @@ DataAccessObject.find = function find(query, options, cb) {
             });
 
             // FIXME: apply "includes" and other transforms - see allCb below
-            memory.all(modelName, ctx.query, options, cb);
+            memory.all(modelName, query, options, cb);
```

Comment:
Oh, now I understand why this change is needed and why it's correct: the `query` is an argument of `queryGeo` which is set to `ctx.query` by the caller. I think we can consider this thread as resolved.